src/svg/timer.svg

```diff
@@ -0,0 +1,5 @@
+<svg width="120" height="120" viewBox="0 0 120 120" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M60.0002 117.6C48.608 117.6 37.4716 114.222 27.9993 107.893C18.5271 101.564 11.1443 92.5676 6.78471 82.0426C2.4251 71.5176 1.28443 59.9361 3.50694 48.7628C5.72945 37.5895 11.2153 27.3262 19.2708 19.2707C27.3263 11.2152 37.5897 5.72931 48.763 3.5068C59.9363 1.28429 71.5177 2.42497 82.0427 6.78457C92.5678 11.1442 101.564 18.5269 107.893 27.9992C114.222 37.4715 117.6 48.6078 117.6 60C117.6 75.2765 111.532 89.9273 100.73 100.729C89.9274 111.531 75.2766 117.6 60.0002 117.6ZM60.0002 13.92C50.8864 13.92 41.9773 16.6226 34.3995 21.6859C26.8217 26.7493 20.9155 33.946 17.4278 42.366C13.9401 50.786 13.0276 60.0512 14.8056 68.9898C16.5836 77.9284 20.9723 86.1391 27.4167 92.5835C33.8611 99.0279 42.0718 103.417 51.0104 105.195C59.949 106.973 69.2142 106.06 77.6342 102.572C86.0542 99.0847 93.2509 93.1785 98.3143 85.6007C103.378 78.0229 106.08 69.1138 106.08 60C106.08 47.7789 101.225 36.0582 92.5836 27.4166C83.942 18.7749 72.2213 13.92 60.0002 13.92V13.92Z" fill="#1E1E1E"/>
```

Comment:
We shouldn't define the fill color, otherwise we won't be able to easily overwrite it. Use `fill="currentColor"` instead